src/Components/test/E2ETest/ServerExecutionTests/GlobalizationTest.cs

```diff
@@ -247,5 +220,21 @@ private static void ReplaceText(IWebElement element, string text)
             element.SendKeys(Keys.Control + "a");
             element.SendKeys(text);
         }
+
+        private void SetCulture(string culture)
+        {
+            var selector = new SelectElement(Browser.FindElement(By.Id("culture-selector")));
+            selector.SelectByValue(culture);
+
+            // Click the link to return back to the test page
+            WaitUntilExists(By.ClassName("return-from-culture-setter")).Click();
+
+            // That should have triggered a redirect, wait for the main test selector to come up.
```

Comment:
This is no longer true right? It's not a redirect with your change.

Reply:
Sure, fixed.